sdk/eventhub/event-hubs/src/eventHubConsumerClientModels.ts

```diff
@@ -71,70 +73,67 @@ export interface PartitionContext {
 }
 
 /**
- * Event handler called when events are received. The `context` parameter can be
- * used to get partition information as well as to checkpoint.
+ * Signature of the user provided function invoked by `EventHubConsumerClient` as and when a set of events is received.
  */
 export type ProcessEventsHandler = (
   events: ReceivedEventData[],
   context: PartitionContext
 ) => Promise<void>;
 
 /**
- * Called when errors occur during event receiving.
+ * Signature of the user provided function invoked by `EventHubConsumerClient` for errors that occur when
+ * receiving events or when executing any of the user provided functions passed to the `subscribe()` method.
  */
-export type ProcessErrorHandler = (error: Error | MessagingError, context: PartitionContext) => Promise<void>;
+export type ProcessErrorHandler = (
+  error: Error | MessagingError,
+  context: PartitionContext
+) => Promise<void>;
 
 /**
- * Called when we first start processing events from a partition.
+ * Signature of the user provided function invoked by `EventHubConsumerClient` just before starting to receive 
+ * events from a partition.
  */
 export type ProcessInitializeHandler = (context: PartitionContext) => Promise<void>;
 
 /**
- * Called when we stop processing events from a partition.
+ * Signature of the user provided function invoked by `EventHubConsumerClient` just before stopping to receive
```

Comment:
```suggestion
 * Signature of the user provided function invoked by `EventHubConsumerClient` just after stopping to receive
```
Does it make more sense to say 'after' instead of 'before' since at this point we won't receive any more events?
I'm worried `before` implies that the user might still be able to receive events.